preview/Win7Msix/Win7MSIXInstaller/MsixRequest.hpp

```diff
@@ -74,6 +81,17 @@ class MsixRequest
         return true;
     }
 
+    bool GetIsCancelClicked()

```

Comment:
>GetIsCancelClicked [](start = 9, length = 18)

Just call it GetIsCancelled or GetIsInstallCancelled to be consistent with the variable name m_isInstallCancelled;

Reply:
Modified method name to GetIsInstallCancelled().